src/pages/RoomOverviewPage.tsx

```diff
@@ -1,25 +1,98 @@
-import { Box, Container, Stack, Typography } from '@mui/material';
+import { Box, Container, Typography } from '@mui/material';
+import Booking from '../booking';
 import BasePage from '../components/BasePage/BasePage';
-import RoomStatusCard from '../components/RoomStatusCard/RoomStatusCard';
-import Timeline from '../components/TimeClock/Timeline';
 import useAlert from '../hooks/useAlert';
+import useBookings from '../hooks/useBooking';
+import { useAllRooms } from '../hooks/useRoom';
+import Room from '../room';
+
+function convertTimestampToTime(timestamp: number) {
+	const date = new Date(timestamp * 1000);
+	const options = {
+		hour: '2-digit',
+		minute: '2-digit',
+	};
+	return date.toLocaleTimeString('da-DK', options);
+}
+
+function renderBookings(allRooms: Room[], bookings: Booking[]) {
```

Comment:
`renderBookings()` virker lidt misvisende, overvej at omdøbe den, evt. `getRoomsWithBookings()`
Kan ikke lige komme på noget bedre selv 🤷‍♂️🍤